sympy/combinatorics/fp_groups.py

```diff
@@ -1712,6 +1726,19 @@ def reidemeister_presentation(fp_grp, H, elm_rounds=2, simp_rounds=2):
     for i in range(20):
         elimination_technique_1(C)
         simplify_presentation(C)
+
+    syms = [g.array_form[0][0] for g in C._schreier_generators]
+    g = free_group(syms)[0]
```

Comment:
It's a free group on generators that have the same symbols as a subset of `C._schreier_free_group`'s generators.